### PR TITLE
parallelized  UI & unit tests in travis [Task #109]

### PR DESCRIPTION
**Pull request by jusleg**
fixes: #109 
### WHAT kind of change does this PR introduce?
Parallelize the builds of travis to run unit and integration test in different threads

### HOW is this accomplished?
Modifying the travis.yml and adding a new script called testing which runs the different testing commands

### Checklist
- [x] Docs have been added / updated to reflect the changes
- [x] I have reviewed and tested the changes :white_check_mark:
- [x] I have squashed my commits to have a reasonable amount of commits
- [x] All of the code I have written adhere to the coding conventions outlined in the coding conventions section of the wiki

### CODE REVIEW

File: testing [hunk 1]
```diff
@@ -0,0 +1,8 @@
+if [[ $1 == integration_tests ]]; then
```
Comment (almiche) [inline testing]: Now thats impressive

**Comment by almiche** [comment]
✅ Styling is adequate and follows coding conventions outlined by the [Kotlin Foundation](https://kotlinlang.org/docs/reference/coding-conventions.html)
  
✅ New methods and functionalities have appropriate tests (testing edge cases + good coverage)

✅ Branch has an acceptable amount of commits with descriptive names and
  * All is well on this front 

✅ PR description matches the code outlined a has mentions to issues

✅ This PR attempts to implement a specific feature (no code blob or epic pr)

Status: **Approved**
Comments: Great work !
